source/API_Reference/Web_API_v3/Tutorials/getting_started_email_activity_api.md

```diff
@@ -0,0 +1,216 @@
+---
+seo:
+  title: Getting Started with the Email Activity API (beta)
+  description: Use the Email Activity query language to get started with the Email Activity API.
+  keywords: email activity, query language, email stats, email events
+title: Getting Started with the Email Activity API (beta)
+weight: 0
+layout: page
+navigation:
+  show: true
+---
+{% warning %}
+Email Activity is in beta and you may not have access - we are currently in the process of rolling out access. For more information, see the [Release Notes]({{root_url}}/release_notes.html#-Email-Activity-Feed-Beta). The feature may also change at any time without notice.
+{% endwarning %}
+
+- [Getting started](#-Getting-started)
+    - [Encoding queries](#-Encoding-queries)
+- [Queries for common use cases](#-Queries-for-common-use-cases)
+- [Query reference](#-Query-reference)
+
+The API gives you access to query all of your stored messages, to query individual messages, and to download a CSV with data about the stored messages.
+
+{% anchor h2 %}
+Getting started
+{% endanchor %}
+
+Start with this basic query to the Email Activity API (replace `<<your API key>>` with an API key from your account):
```

Comment:
+Start with this basic query to the Email Activity Feed API (replace `<<your API key>>` with an API key from your account):